src/work-allocation/services/session-storage.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
```

Comment:
Should this be moved to common-library at some point?

Reply:
If it's going to be used elsewhere in MC, it should be moved to a shared area within MC. If it's going to be used in one of the other applications, the common library makes sense.

For now, though, this is the only use of session storage I've found, which is why it's sitting where it is. :)